actions/responses.go

```diff
@@ -3,17 +3,20 @@ package actions
 import (
 	"net/http"
 
-	mbvalidator "github.com/mailbadger/app/validator"
-
 	"github.com/gin-gonic/gin"
-	"gopkg.in/go-playground/validator.v9"
+	"github.com/go-playground/validator/v10"
+
+	mbvalidator "github.com/mailbadger/app/validator"
 )
 
 func AbortWithError(c *gin.Context, err error) {
-	for _, fieldErr := range err.(validator.ValidationErrors) {
+	if fieldErrors, ok := err.(validator.ValidationErrors); ok {
 		c.JSON(http.StatusBadRequest, gin.H{
-			"message": mbvalidator.FieldError{Err: fieldErr}.String(),
+			"message": fieldErrors.Error(),
```

Comment:
Maybe we should have a message and errors object in the response, something like this:

```json
{
  "message": "Invalid parameters, please try again.",
  "errors": {
    "key1": "Validation failed on condition: required",
    "key2": "Invalid email format"
  }
}
```